packages/types/src/lib/interfaces.ts

```diff
@@ -1055,34 +1081,35 @@ export interface GetSignSessionKeySharesProp {
   body: SessionRequestBody;
 }
 export interface CommonGetSessionSigsProps {
+  // If you want to pass the `authNeededCallback`, having a PKP public key is necessary. This can be used if you do not have a wallet and wish to authenticate with your PKP.
```

Comment:
> If you want to pass the `authNeededCallback`, having a PKP public key is necessary.

I don't think is true?

Reply:
It's not, got that idea from [here](https://v6-api-doc-lit-js-sdk.vercel.app/interfaces/types_src.GetSessionSigsProps.html#authNeededCallback). Might need to rewrite it as well

Reply:
Changed this